tests/cluecode/data/plugin_email_url/urls-threshold.expected.json

```diff
@@ -1,4 +1,6 @@
 {
+  "license_references": [],
+  "rule_references": [],
```

Comment:
Rule references is not an obvious, non-ambiguous name for a top level attributes. It would have to be `license_rule_references` or in any case something that very explicitly is about licenses

Reply:
Also for non-license CLI tests use the --no-license-references option this way the expectation will not change